homework1/homework1.css

```diff
@@ -0,0 +1,17 @@
+* {margin: 0; padding: 0;}
+body {
+    background-color: blue;
+}
+
+.one {
+    text-align: center;
+    align-items: center;
+}
+h3 {
+    color: white;
+    font-family: segoe ui;
+    font-size: 13px;
+}
+button {
+    margin-top: 10px;
+}
```

Comment:
1 tab should be 2 spaces (ref: https://www.kindacode.com/article/vs-code-how-to-change-indentation-2-spaces-4-spaces/) - Chỉnh lại format 1 tab tương ứng 2 space để code cho gọn lại nhé